tools/packaging/scripts/configure-hypervisor.sh

```diff
@@ -445,6 +445,9 @@ generate_qemu_options() {
 	qemu_options+=(functionality:--enable-cap-ng)
 	qemu_options+=(functionality:--enable-seccomp)
 
+	# Support NUMA topology
+	qemu_options+=(functionality:--enable-numa)
```

Comment:
Should that be the default? What is the cost in terms of qemu size? We have put a lot of efforts into trimming down qemu and the kernel for kata, so maybe adding back NUMA should only be done for those who need it?

For k8s, maybe this would end up as a different runtime class, kata-numa?

Reply:
No let's make this configurable as well that is tight to the kernel config.